certified-connectors/PowerBI Robots/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Account Id",
+          "description": "The Account Id for this api",
+          "tooltip": "Provide your Account Id",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0577bd",
+    "capabilities": [],
+    "publisher": "Ant\u00f3nio Arantes"
```

Comment:
Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in English language.

Reply:
commited on Identation, response and missing properties corrections